operate/client/src/App/ProcessInstance/BottomPanel/VariablePanel/index.tsx

```diff
@@ -27,6 +38,22 @@ const VariablePanel = observer(function VariablePanel() {
     };
   }, [processInstanceId]);
 
+  useEffect(() => {
+    if (flowNodeId) {
+      fetchListeners({
+        fetchType: 'initial',
+        processInstanceId: processInstanceId,
+        payload: {flowNodeId},
+      });
+    }
+  }, [fetchListeners, processInstanceId, flowNodeId]);
+
+  useEffect(() => {
+    listeners?.length > 0
+      ? setFlowNodeHasListeners(true)
+      : setFlowNodeHasListeners(false);
+  }, [listeners]);
```

Comment:
As an alternative: Something like ´hasFlowNodeListeners´ could also be a computed value in the store.